<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClient</class>
 <widget class="QDialog" name="TcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>49</y>
     <width>36</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>20</y>
     <width>36</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>IP：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientIPlineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>21</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>10.4.0.78</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientPortlineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>5555</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKpushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确 定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
